Embedded Metadata.js

```diff
@@ -1383,12 +1366,12 @@ var testCases = [
 					}
 				],
 				"date": "2013",
-				"abstractNote": "Signaling data from the cellular networks can provide a means of analyzing the efficiency of a deployed transportation system and assisting in the formulation of transport models to predict its fut ...",
+				"abstractNote": "DiVA portal is a finding tool for research publications and student theses written at the following 47 universities and research institutions.",
```

Comment:
why is it picking up the site description from og:description rather than the abstract from DC.description? Since this is also true for the current EM translator, I'll accept this and create a new issue.

Reply:
https://github.com/zotero/translators/issues/1761